Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,34 @@ label monika_enjoyingspring:
     else:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_unknown",category=['psychology'],prompt="Fear of the unknown",random=True))
+
+label monika_unknown:
+    m 2euc "Hey, [player]..."
+    m 3eud "Did you know that a lot of people are afraid of the dark?"
+    m 3eka "Even though it's often dismissed as a childish fear, it's not that uncommon for adults to suffer from it as well."
+    m 4eub "Fear of the dark, called 'nyctophobia', is usually caused by the mind's exaggerated guess of what may be hiding in the shadows, rather than darkness itself."
+    m 4eua "We're scared because we don't know what's there...{w=1}even if it's usually nothing."
+    m 3eka "...And I'm not just talking about monsters under the bed, or menacing silhouettes...{w=1} Try moving in a dark room."
+    m 3eub "You'll find that you're instinctively being more careful of where you step so you don't hurt yourself."
+    m 3esd "It makes sense;{w=0.5} humans have learned to be wary of the unknown in order to survive."
+    m 3esc "You know, like being cautious around strangers, or thinking twice before jumping into unfamiliar situations."
+    m 3dsd "'{i}Better the devil you know than the devil you don't.{/i}'"
+    m 3rksdlc "But even if that frame of thinking has helped people survive for hundreds of thousands of years, I think it can also do a lot of harm nowadays."
+    m 1rksdld "Like how some people are unsatisfied with their jobs but are too afraid to quit..."
+    m 1eksdlc "Most of them can't afford to lose their source of income, so resigning isn't an option."
+    m 3rksdlc "Plus, having to go through interviews again, finding a job that pays enough, changing your routine..."
+    m 3ekd "It just seems like it's easier being miserable because it's more comfortable,{w=0.5} even if they'd be much happier in the long run."
```

Comment:
I get what you're saying, but "easier being miserable because it's more comfortable" doesn't sound right to me. 

I think there could be a better way to word it. 

Maybe you could instead say something about how that's hassle and a lot of work when you don't even know how it'll turn out in the end. Hence the topic, Fear of the unknown.

Reply:
True. (Working on it)

Reply:
After giving it some thought, I couldn't come up with a phrasing that isn't too long and keeps the feeling I was going for. If you do have a particular change in mind, do suggest it, otherwise I'll leave it as is.

Reply:
I don't know if you think this is too long or fits the feeling you were going for, but my idea adds an extra line:

m 3ekd "It just seems like it's more comfortable to stick with it, even if it makes you miserable."
m 1rksdlc "You'd be much happier in the long run otherwise, but it's that unknown aspect to it;{w=0.5} you don't know how it'll turn out."

Reply:
I think that ends up being too long-winded for such a simple point. It'd also end up being too repetitive, as it's already implied in this line further below that there's no guarantee it'll be worth it:

`m 1rkc "...Not that making that kind of decision is easy, or even safe."`

I think I'll leave it as is, but I do appreciate your suggestion.

Reply:
Alright, that's fair. I think this topic turned out pretty nice and I'm looking forward to hopefully see it make it in.

Reply:
Thanks, me too. ~~Also is it me or is your meme not visible unless you go to edits?~~

Reply:
~~What meme~~